Changelog.minor.md

```diff
@@ -11,6 +11,7 @@ release the new version.
 
 ### Added
 
+-   #1170: Check and offer JLink install on update check.
 -   #1147: Automatic release candidates.
```

Comment:
```suggestion
-   #1170: Mechanism for checking SEGGER J-Link installation update.
-   #1147: Mechanism for automatic release candidates.
```

Reply:
I would assume that this doesn't require correction as it is only for us developers to track changes.
I'll still apply them just in case though.